src/Modules/OrchardCore.Commerce/Assets/Styles/shopping-cart-widget.scss

```diff
@@ -0,0 +1,19 @@
+.widget-shopping-cart-widget {
+    position: absolute;
+    right: 10px;
+    top: 10px;
+}
+
+.shopping-cart-link {
+    position: relative;
+
+    // Making sure widget is always visible.
+    z-index: 9999;
+}
+
+.item-count {
+    border-radius: 50%;
+    width: 35px;
+    height: 35px;
+    font-size: 25px;
```

Comment:
If the intention is to make font-size smaller than the circle size then it should be calculated,
e.g.`calc(var(cart-widget-circle-size) - 10px)`